src/CodeGen_Hexagon.h

```diff
@@ -24,8 +24,8 @@ class CodeGen_Hexagon : public CodeGen_Posix {
 
     void init_module();
 
-    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer);
-    void pop_buffer(const std::string &name, int dimensions);
+    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer, bool global = false);
```

Comment:
EDIT: I hadn't yet read the part where the base definition was changed too. (Side note: the 'override' keyword is nice, maybe we should use it :-)